bp/src/BPConfiguration.m

```diff
@@ -140,12 +140,36 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
             "A .GlobalPreferences.plist simulator preferences file to be copied to any newly created simulators before booting"},
     {363, "script-file", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "scriptFilePath",
         "A script that will be called after the simulator is booted, but before tests are run. Can be used to do any setup (e.g. installing certs). The environment will contain $BP_DEVICE_ID with the ID of the simulator and $BP_DEVICE_PATH with its full path. Exit with zero for success and non-zero for failure."},
+    {364, "test-plan-path", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testPlanPath", "The path of a json file which describes the test plan. It's a equivalent of .xctestrun file generated by an external build system, e.g. Bazel"},
```

Comment:
```suggestion
    {364, "test-plan-path", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testPlanPath", "The path of a json file which describes the test plan. It is equivalent to the .xctestrun file generated by Xcode, but it can be generated by a different build system, e.g. Bazel"},
```